runner/status.go

```diff
@@ -94,6 +96,7 @@ func (p RunStatus) String() string {
 		s += fmt.Sprintf(" # Error: %s", p.Error)
 	}
 	s += fmt.Sprintf(" # Stdout: %s # Stderr: %s", p.StdoutRef, p.StderrRef)
+	// TODO: determine how a RunStatus with a non-nil BazelResult should look
```

Comment:
I just added this for runner.Commands (see runner/runner.go - doesn't add anything when nil), that's what the String() funcs in bazelapi are for

Reply:
Gotcha. The duplication of fields is a little weird though. There ends up being 2 ExitCodes and a StdoutRef, StdoutDigest, and StdoutRaw (+ errs) for a RunStatus with an ActionResult

Reply:
Sure - could add an additional short/truncated string function to bazelapi and use that here